apps/src/lib/util/javascriptMode.js

```diff
@@ -19,11 +19,13 @@ export function injectErrorHandler(handler) {
 
 /** @see JavaScriptModeErrorHandler#outputError */
 export function outputError(...args) {
+  console.log('calling outputError  ' + args);
```

Comment:
Can we remove this and another `console.log` statement below before merging?

Reply:
Thanks for catching those! Just removed.